### PR TITLE
Add Capybara Exchange TVL Adapter

### PR DESCRIPTION
**Pull request by thanh-wombat**
Related PR: https://github.com/DefiLlama/dimension-adapters/pull/1527

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/capybara-exchange exports TVL: 
        
 
 
    klaytn                    614.24 k
    
    total                    614.24 k

File: projects/capybara-exchange/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const config = require("./config");
+const { sumTokens2 } = require("../helper/unwrapLPs");
+
+Object.keys(config).forEach((chain) => {
+  const arg = config[chain];
+  module.exports[chain] = {
+    tvl: async (_, _b, { [chain]: block }, { api }) => {
```
Comment (g1nt0ki) [inline projects/capybara-exchange/index.js:7]: mostly good, change this line to `tvl: async (api ) => {`

**Comment by llamatester** [comment]
The adapter at projects/capybara-exchange exports TVL: 
        
 
 
    klaytn                    603.14 k
    
    total                    603.14 k

File: projects/capybara-exchange/index.js (hunk 1 above)
Comment (thanh-wombat) [reply projects/capybara-exchange/index.js]: Thanks! I followed your suggestion. Can you take a look again?

**Comment by Define101** [comment]
@thanh-wombat can you provide website link pls

**Comment by thanh-wombat** [comment]
Thanks! The link is: https://www.capybara.exchange/

**Pull request merged**